src/plone/restapi/tests/test_types.py

```diff
@@ -130,14 +130,14 @@ def test_get_jsonschema_for_portal_type(self):
         )
         self.assertEqual(jsonschema['title'], 'Page')
         self.assertEqual(jsonschema['type'], 'object')
-        self.assertIn('title', jsonschema['properties'].keys())
+        self.assertIn('title', list(jsonschema['properties'].keys()))
```

Comment:
this should be written: `self.assertIn('title', list(jsonschema['properties']))`